pkg/dsl/process/handle_test.go

```diff
@@ -41,7 +41,7 @@ image: "myserver"
 	myAppName, err := ctxInst.Lookup("context", "appName").String()
 	assert.Equal(t, nil, err)
 	assert.Equal(t, "myapp", myAppName)
-	inputJs, err := ctxInst.Lookup("context", "input").MarshalJSON()
+	inputJs, err := ctxInst.Lookup("context", "output").MarshalJSON()
```

Comment:
please use a const variable

Reply:
by the way, please add a test for it. It seems we change from `input` to `output` but no test broken. That could be hard to maintain

Reply:
OutputFieldName